.github/workflows/pypipublish.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will publish a package to PyPi when a pull request is merged.
+# The version of the package must be set in `package.json`.
+
+name: PyPi Package
+
+on:
+  push:
+    branches: master
+
+jobs:
+  publish-pypi:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+      - name: Build package
+        run: |
+        python setup.py sdist bdist_wheel
```

Comment:
I think this will break right now as I'm pretty sure that the build step will still require using nodejs.

The jupyterlab-git version is probably good to base this off of https://github.com/jupyterlab/jupyterlab-git/blob/master/.github/workflows/publish.yml